### PR TITLE
Add UTs for Netflow CRD

### PR DESCRIPTION
**Pull request by tanyatukade**
- Added UTs for Netflow CRD to increase coverage

### CODE REVIEW

File: pkg/controller/netflow.go [hunk 1]
```diff
@@ -80,15 +80,16 @@ func (cont *AciController) initNetflowInformerBase(listWatch *cache.ListWatch) {
...
 	netflowPolicy := obj.(*netflowpolicy.NetflowPolicy)
 	key, err := cache.MetaNamespaceKeyFunc(netflowPolicy)
 	if err != nil {
 		NetflowPolicyLogger(cont.log, netflowPolicy).
 			Error("Could not create key:" + err.Error())
-		return
+		return false
 	}
 	cont.queueNetflowUpdateByKey(key)
+	return false
```
Comment (tomflynn) [inline pkg/controller/netflow.go:92]: Shouldn't this return true? Right now it always returns false...

File: pkg/controller/netflow.go [hunk 2]
```diff
@@ -106,44 +107,38 @@ func (cont *AciController) queueNetflowUpdate(netflowpolicy *netflowpolicy.Netfl
...
 	}
 	nfkey, err := cache.MetaNamespaceKeyFunc(nf)
 	if err != nil {
 		NetflowPolicyLogger(cont.log, nf).
 			Error("Could not create netflow key: ", err)
-		return
+		return false
 	}
 	cont.apicConn.ClearApicObjects(cont.aciNameForKey("nfp", nfkey))
+	return false
```
Comment (tomflynn) [inline pkg/controller/netflow.go:133]: same comment...you're always returning false regardless of whether there's a problem

**Review by tomflynn — CHANGES_REQUESTED** [review]

File: pkg/controller/netflow.go (hunk 1 above)
Comment (tanyatukade) [reply pkg/controller/netflow.go]: You are right! I have addressed your comments in the commit below.

Comment (jayaramsatya) [reply pkg/controller/netflow.go]: you don't need a return bool value here.  make it void function you are not handling any error condition here.

Comment (tanyatukade) [reply pkg/controller/netflow.go]: changed it.

**Review by tomflynn — APPROVED** [review]

**Pull request merged**